app/process/model/transaction/transaction.go

```diff
@@ -0,0 +1,25 @@
+package transaction
+
+type (
+	Transaction struct {
```

Comment:
I wonder if placing `Hedera` in front of the models would be a good fit, enabling people reading realizing that this is Hedera specific. Ethereum transactions would have a much different format.